lib/model.js

```diff
@@ -98,6 +98,34 @@ var stringUtils = require('underscore.string');
 
 var Model = module.exports = registry.modelBuilder.define('Model');
 
+function getRemotingType(prop) {
+  if (prop.type === Number) {
+    return 'number';
+  }
+  if (prop.type === Boolean) {
+    return 'number';
```

Comment:
@raymondfeng please add a unit-test for this case, to make sure we don't introduce a similar bug in the future.